modules/gitops-release-notes-1-1.adoc

```diff
@@ -0,0 +1,129 @@
+
+
+[id="breaking-change-1-1_{context}"]
+== Breaking Change
+=== Upgrading from {gitops-title} v1.0.1
+
+When you upgrade from {gitops-title} `v1.0.1` to `v1.1`, the {gitops-title} Operator renames the default Argo CD instance created in the `openshift-gitops` namespace from `argocd-cluster` to `openshift-gitops`.
+
+This is a breaking change and needs the following steps to be performed manually, before the upgrade:
+
+. Go to the {product-title} web console and copy the content of the `argocd-cm.yml` config map file in the `openshift-gitops` namespace to a local file. The content may look like the following example:
++
+.Example argocd config map YAML
+[source,yaml]
+----
+    kind: ConfigMap
+    apiVersion: v1
+    metadata:
+     selfLink: /api/v1/namespaces/openshift-gitops/configmaps/argocd-cm
+     resourceVersion: '112532'
+     name: argocd-cm
+     uid: f5226fbc-883d-47db-8b53-b5e363f007af
+     creationTimestamp: '2021-04-16T19:24:08Z'
+     managedFields:
+    ...
+     namespace: openshift-gitops
+     labels:
+       app.kubernetes.io/managed-by: argocd-cluster
+       app.kubernetes.io/name: argocd-cm
+       app.kubernetes.io/part-of: argocd
+       data:                                            "" <1>
+        admin.enabled: 'true'
+        statusbadge.enabled: 'false'
+        resource.exclusions: |
+          - apiGroups:
+            - tekton.dev
+            clusters:
+            - '*'
+            kinds:
+            - TaskRun
+            - PipelineRun
+        ga.trackingid: ''
+        repositories: |
+          - type: git
+            url: https://github.com/user-name/argocd-example-apps
+        ga.anonymizeusers: 'false'
+        help.chatUrl: ''
+        url: >-  https://argocd-cluster-server-openshift-gitops.apps.dev-svc-4.7-041614.devcluster.openshift.com   "" <2>
+        help.chatText: ''
+        kustomize.buildOptions: ''
+        resource.inclusions: ''
+        repository.credentials: ''
+        users.anonymous.enabled: 'false'
+        configManagementPlugins: ''
+        application.instanceLabelKey: ''
+----
+<1> Restore only the _data_ section of the content in the `argocd-cm.yml` config map file manually.
+<2> Replace the URL value in the config map entry with the new instance name `openshift-gitops`.
+
+. Delete the default `argocd-cluster` instance.
+. Edit the new `argocd-cm.yml` config map file to restore the entire _data_ section manually.
+. Replace the URL value in the config map entry with the new instance name `openshift-gitops`. For example, in the preceding example, replace the URL value with the following URL value:
++
+[source,yaml]
+----
+        url: >-  https://openshift-gitops-server-openshift-gitops.apps.dev-svc-4.7-041614.devcluster.openshift.com
```

Comment:
We can remove the extra space before the `url` field here.
Also, I am not sure why this is highlighted in red in the preview, here as well as in the YAML snippet above.

Reply:
Done